TransIT.BLL/Services/ImplementedServices/UserService.cs

```diff
@@ -99,7 +99,7 @@ public virtual async Task<User> UpdateAsync(User model)
                     await _unitOfWork.SaveAsync();
                     return res;
                 }
-
+                model.Password = _hasher.HashPassword(model.Password);
```

Comment:
That's bad! We must have separate endpoint for password updating
`POST api/v1/users/password`
And create separate method in UserService for that!